packages/fxa-auth-server/test/local/payments/iap/iap-config.js

```diff
@@ -128,10 +128,7 @@ describe('IAPConfig', () => {
         await iapConfig.plans('testApp');
         assert.fail('Expected exception thrown.');
       } catch (err) {
-        assert.strictEqual(
-          err.message,
-          'IAP Plans document does not exist for testApp'
-        );
+        assert.strictEqual(err.message, 'Unknown subscription');
```

Comment:
Curious why the message is `Unknown subscription` and error is called `Unknown App Name`?

Reply:
Got confirmation from Ben, it should be as you suggested. I'll make the update and squash my commits - thanks!